File: from-3301-to-3400/3340-check-balanced-string/solution.py
class Solution:
    def isBalanced(self, num: str) -> bool:
        evenSum, oddSum = 0, 0
        for i, d in enumerate(num):
            if i % 2:
                oddSum += int(d)
            else:
                evenSum += int(d)
        return evenSum == oddSum

File: from-3301-to-3400/3340-check-balanced-string/solution.go
func isBalanced(num string) bool {
    evenSum, oddSum := 0, 0
    var digit int
    for i := 0; i < len(num); i ++ {
        digit = int(num[i] - '0')
        if i % 2 == 0 {
            evenSum += digit
        } else {
            oddSum += digit
        }
    }
    return evenSum == oddSum
}